site/docs/latest/clients/WebSocket.md

```diff
@@ -81,6 +81,19 @@ The producer endpoint requires you to specify a {% popover property %}, {% popov
 
 {% endpoint ws://broker-service-url:8080/ws/producer/persistent/:property/:cluster/:namespace/:topic %}
 
+##### Query param
+
+Key | Type | Required? | Explanation
+:---|:-----|:----------|:-----------
+`sendTimeoutMillis` | string | no | Send timeout (default: 30 secs)
```

Comment:
type `int`

Reply:
oops.. I missed to change data-type everywhere.. :-(